python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -248,10 +243,30 @@ def _create_chat_message_content(self, response: Mapping[str, Any], metadata: di
         )
 
     def _create_streaming_chat_message_content(
-        self, part: Mapping[str, Any], metadata: dict[str, Any]
+        self, part: Mapping[str, Any] | ChatResponse, metadata: dict[str, Any]
```

Comment:
Question: when will it return a mapping and when will it return a ChatResponse?

Reply:
the chatresponse was introduced in their SDK, this keeps it compatible with both for the moment